ui/desktop/src/components/settings_v2/extensions/agent-api.ts

```diff
@@ -1,7 +1,7 @@
 import { ExtensionConfig } from '../../../api/types.gen';
 import { getApiUrl, getSecretKey } from '../../../config';
 import { toastService, ToastServiceOptions } from '../../../toasts';
-import { replaceWithShims } from './utils';
+import { replaceWithShim } from '../../../utils/shims';
```

Comment:
I've renamed the function to use `shim` singular since it is only replacing a single command with a single shim on each call.